File: android/app/src/main/java/com/texasgamer/zephyr/util/BuildConfigUtils.java
package com.texasgamer.zephyr.util;

import com.texasgamer.zephyr.BuildConfig;

/**
 * BuildConfig utilities.
 */
public final class BuildConfigUtils {

    private BuildConfigUtils() {
    }

    public static int getVersionCode() {
        return BuildConfig.VERSION_CODE;
    }

    public static String getVersionName() {
        return BuildConfig.VERSION_NAME;
    }

    public static String getPackageName() {
        return BuildConfig.APPLICATION_ID;
    }

    public static String getFlavor() {
        return BuildConfig.FLAVOR;
    }

    public static String getBuildType() {
        return BuildConfig.BUILD_TYPE;
    }

    public static boolean isDebug() {
        return BuildConfig.DEBUG;
    }
}

File: android/app/src/main/java/com/texasgamer/zephyr/util/BuildConfigUtils.kt
package com.texasgamer.zephyr.util

import com.texasgamer.zephyr.BuildConfig

/**
 * BuildConfig utilities.
 */
object BuildConfigUtils {
    @JvmStatic
    val versionCode: Int
        get() = BuildConfig.VERSION_CODE

    val versionName: String
        get() = BuildConfig.VERSION_NAME

    @JvmStatic
    val packageName: String
        get() = BuildConfig.APPLICATION_ID

    val flavor: String
        get() = BuildConfig.FLAVOR

    val buildType: String
        get() = BuildConfig.BUILD_TYPE

    val isDebug: Boolean
        get() = BuildConfig.DEBUG
}